### PR TITLE
Created first pull request Voronkova

### PR DESCRIPTION
**Pull request by gallinavoronkova**

### CODE REVIEW

File: src/test/java/e2e/CreateContactViaApiEditContactViaUiTest.java [hunk 1]
```diff
@@ -25,5 +25,7 @@ public void createContactViaApiEditContactViaUiTest() {
         contactPage = new ContactPage(app.driver);
         contactPage.waitForLoading();
         contactPage.openContactById(String.valueOf(id));
+
```
Comment (LeonadO555) [inline src/test/java/e2e/CreateContactViaApiEditContactViaUiTest.java:28]: Need to remove

Comment (LeonadO555) [reply src/test/java/e2e/CreateContactViaApiEditContactViaUiTest.java:28]: We did it on the lesson

Comment (gallinavoronkova) [reply src/test/java/e2e/CreateContactViaApiEditContactViaUiTest.java:28]: We did it on the lesson

Comment (LeonadO555) [reply src/test/java/e2e/CreateContactViaApiEditContactViaUiTest.java:28]: okay

Comment (LeonadO555) [inline src/test/java/e2e/CreateContactViaApiEditContactViaUiTest.java:28]: 123

**Review by LeonadO555 — CHANGES_REQUESTED** [review]

**Review by LeonadO555 — APPROVED** [review]

**Pull request merged**